source/includes/kubernetes/_k8_daemonsets.md

```diff
@@ -142,6 +142,84 @@ Create a daemon set in a given [environment](#administration-environments).
 | `spec.spec`<br/>*object*                   | The specification used to create and run the pod(s) within the daemon set. |
 
 
+| Optional Attributes                        | &nbsp;                                                                     |
+| ------------------------------------------ | -------------------------------------------------------------------------- |
+| `kind`<br/>_string_                        | The string value of the REST resource that this object represents.         |
```

Comment:
Should we someone try to also mention that this maps to a Kubernetes resource kind too? Would it be helpful?
```
The string value of the REST resource that this object represents. (It is also the same as the Kubernetes resource kind)
```

Reply:
Yeah, that could be valuable. I based it off of what was in the section for create, so should that be updated, too?

Reply:
Also for this specific API this can only ever be `DaemonSets` right? If I can this specific API with some other value for kind then it would fail I assume?

Reply:
I feel like it might be useful. What do you think?

@m-orfanos @Gerudhoh - what you guys think about ☝️ ?

Reply:
The validations would fail for sure if the `kind` doesn't match the kind of the existing resource, and I imagine the dry run would fail if you tried to create a daemon set with `kind: Ingress` or something like that.

Reply:
I think, for API users we can completely remove this field. It is inferred from the API endpoint `/daemonsets` and it's always going to be constant

Reply:
Since I think most of the docs have `kind` in them, would it make sense to put that as a separate issue to resolve in all the docs all at once?

Reply:
True. That makes sense, instead of removing in one place 🙌